### ISSUE TITLE
Neoforge support

### ISSUE THREAD

**Issue by mxnmnm** [BlueDuckYT/The-Outer-End#90]
The version restrictions you have added for forge make it incompatible with neoforge :(

**Comment by mxnmnm** [comment]
This specifically happens on 1.0.7 and previous versions (at least 1.0.5 and below, didn't test 1.0.6) worked so I see no reason to not add neoforge's latest version under the mandatory versions.

**Comment by BlueDuckYT** [comment]
On curseforge it is set to allow neoforge. Is it an issue on loading?

**Comment by mxnmnm** [comment]
Yes it is an issue loading because of the forge restrictions you set within the mod itself.

**Comment by BlueDuckYT** [comment]
Darn, I’ll fix that tonight

**Comment by mxnmnm** [comment]
This is still an issue, you need to edit the mods.toml to allow neoforge's versions

**Comment by ianm1647** [comment]
you set the forge version to a higher version than what neoforge is 

forge is 47.2.2, and the latest neoforge version is 47.1.101

**Comment by BlueDuckYT** [comment]
Alright, I’ll put out a patch tomorrow. Can’t guarantee everything will work perfectly though

**Comment by ianm1647** [comment]
all you'd have to do is rollback to before you made the changes after updating to forge 47.2.2 and publish the mod with the changes and fixes you made AFTER you updated it but staying on the older forge version. i know some of those changes were made due to the forge update, but the major changes like the vertical planks being removed and such should be easy.

**Comment by BlueDuckYT** [comment]
1.0.9 should now run on neoforge
